tests/functional/aws-node-sdk/test/legacy/testBucketVersioning.js

```diff
@@ -0,0 +1,131 @@
+import assert from 'assert';
```

Comment:
We have `./tests/functional/aws-node-sdk/test/bucket` already created, you can add the file there.